pkg/backporter/backporter.go

```diff
@@ -278,94 +304,218 @@ func GetBugHandler(client bugzilla.Client) HandlerFuncWithErrorReturn {
 	}
 }
 
+func getClonesTemplateData(bugID int, w http.ResponseWriter, client bugzilla.Client, allTargetVersions sets.String) (*ClonesTemplateData, error) {
+	bug, err := client.GetBug(bugID)
+	if err != nil {
+		w.WriteHeader(http.StatusNotFound)
+		wpErr := writePage(w, "Not Found", errorTemplate, fmt.Sprintf("Bug#%d not found", bugID))
+		if wpErr != nil {
+			http.Error(w, "Error building page.", http.StatusInternalServerError)
+			return nil, wpErr
+		}
+		return nil, err
+	}
+	prs, err := client.GetExternalBugPRsOnBug(bug.ID)
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		wpErr := writePage(w, "Error.", errorTemplate, fmt.Sprintf("Bug#%d - error occured while retreiving list of PRs : %v", bug.ID, err))
+		if wpErr != nil {
+			http.Error(w, "Error building page.", http.StatusInternalServerError)
+			return nil, wpErr
+		}
+		return nil, err
+	}
+	parent, err := client.GetRootForClone(bug)
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		wpErr := writePage(w, "Error.", errorTemplate, fmt.Sprintf("Bug#%d Details of parent could not be retrieved : %v", bug.ID, err))
```

Comment:
tbh I find it a bit weird to have a page titled `Error.` rather than just `Error`? I don't think its common for titles anywhere to have trailing punctuation?

Reply:
Yeah, you are right

Reply:
Although I do feel that Error! is more appropriate

Reply:
That feels to me like windows 98 `Error! no work!`